app/views/comments/edit.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="row static-width main-content-container">
+
+  <div class="col-md-7 col-md-offset-1">
+    <%= render partial: 'layouts/errors', locals: { element: @comment } %>
+
+    <ul class="no-list-style no-padding">
+      <%= form_for [@comment.commentable, @comment], url: comment_path(@comment) do |t| %>
+        <li><%= t.label :message, class: "form-field" %></li>
+        <li><%= t.text_area :content, class: "form-field content-field" %></li>
+        <li><%= submit_tag("Submit Your Comment", class: "form-button") %></li>
+      <% end %>
+    </ul>
+
+  </div>
+
+  <div class="col-md-3">
+    <div class="col-md-12 question-meta">
+      <h1 class="question-meta-header">How to Ask</h1>
```

Comment:
It looks like these markdown tips are repeated a few places - good partial candidate.